lib/common.js

```diff
@@ -154,11 +154,13 @@ function stringifyRequest(options, body) {
   let { port } = options
   // TODO-coverage: Add a test to cover the missing condition, or remove if
   // not reachable.
-  if (!port) port = options.proto == 'https' ? '443' : '80'
+  // The step seems useless considering we don't show the port for standard 80/443 anyway.
```

Comment:
👍 Seems like there's a good chance this was copied and pasted from somewhere and that it could be refactored.

Also note that there's some coverage missing. I think the line `port = options.proto == 'https' ? '443' : '80'` is not currently run by tests.